structured/structured.go

```diff
@@ -80,9 +80,14 @@ func (desc *TableDescriptor) AllocateIDs() error {
 				index.ColumnIDs[j] = columnNames[colName]
 			}
 		}
+
 		// Mildly confusing: index is not a pointer so we need to set it back into
```

Comment:
No. Once the structs are updated they need to update the original struct they came from.